library/std/src/sync/mutex.rs

```diff
@@ -229,6 +230,21 @@ impl<T> Mutex<T> {
         }
         m
     }
+
+    /// Creates a new mutex in an unlocked state ready for use, and wraps it in an [`Arc`].
```

Comment:
Sounds complicated. Why not just say this is a shorthand for `Arc::new(Mutex::new(t))` or this creates `Arc<Mutex<T>>`?